### DISCUSSION TITLE
how to return "Last-modified" http header in responses

### DISCUSSION THREAD

**Discussion by cgoines** [cantaloupe-project/cantaloupe#569] [Q&A] [answered]
Hello,

I have the cache-control header enabled in my cantaloupe.properties, but i was wondering why cantaloupe doesn't return the "last-modified" http header for caching and what property needs to be set to return that if possible?

thanks!
-c

**Comment by adolski** [comment, marked as answer]
My memory is hazy about why there is no `Last-Modified` header. No one else has ever brought it up. But it would be useful to have. I'll put it on the to-do list for 6.0.
